### PR TITLE
`Development`: Fix console.error logs during client test execution

### PR DESCRIPTION
**Pull request by krusche**
#### Client
- [x] I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client/) and ensured that the layout is responsive.

### Motivation and Context
This PR gets rid of nasty error messages due to missing imports leading to console errors

### Review Progress
No testing is needed, because we only change client tests and potentially remove console.error in the client production code

#### Code Review
- [x] Review 1
- [x] Review 2

### CODE REVIEW

File: src/main/webapp/app/exercises/modeling/manage/modeling-exercise.service.ts [hunk 1]
```diff
@@ -77,10 +77,11 @@ export class ModelingExerciseService implements ExerciseServicable<ModelingExerc
             .pipe(map((response: HttpResponse<ModelingPlagiarismResult>) => response.body!));
     }
 
-    convertToPdf(model: string, filename: string): Observable<any> {
+    convertToPdf(model: string, filename: string): Observable<HttpResponse<Blob>> {
         return this.http
             .post(`${SERVER_API_URL}api/apollon-convert/pdf`, { model }, { observe: 'response', responseType: 'blob' })
-            .pipe(map((response: HttpResponse<Blob>) => downloadStream(response.body, 'application/pdf', filename)));
+            .pipe(tap((response: HttpResponse<Blob>) => downloadStream(response.body, 'application/pdf', filename)))
+            .pipe(tap((foo) => foo));
```
Comment (Strohgelaender) [inline src/main/webapp/app/exercises/modeling/manage/modeling-exercise.service.ts:84]: Why is this needed? Seems a bit out of place to me.

Comment (julian-christl) [reply src/main/webapp/app/exercises/modeling/manage/modeling-exercise.service.ts:84]: Whoopsie. Looks like I pushed a debugging thingy

**Review by julian-christl — APPROVED** [review]
I reviewed the rest I didn't touch (which is most of it), and it looks good

**Review by ge65cer — APPROVED** [review]
Looks good, the tests passed!
